includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```
@@ -1093,19 +1095,10 @@ public function payment_fields() {
-		if ( ! WC_Stripe_Feature_Flags::is_checkout_sessions_available() ) {
-			return;
-		}
-
-		// Bail if not on the checkout page.
-		if ( ! is_checkout() ) {
-			return;
-		}
-
-		echo '<div id="wc-stripe-currency-selector" class="wc-stripe-currency-selector" style="margin: 12px 0;"></div>';
+		wc_deprecated_function( __METHOD__, '10.6.0' );
 	}
```

PHPMD (2.15.0)

[error] The method attach_currency_selector_element is not named in camelCase. (undefined)

(CamelCaseMethodName)